topostats/tracing/disordered_tracing.py

```diff
@@ -0,0 +1,708 @@
+"""Generates disordered traces (pruned skeletons) and metrics."""
+
+from __future__ import annotations
+
+import logging
+import warnings
+
+import numpy as np
+import numpy.typing as npt
+import skimage.measure as skimage_measure
+from scipy import ndimage
+from skimage.filters import gaussian, threshold_otsu
+from skimage.morphology import label
+
+from topostats.logs.logs import LOGGER_NAME
+from topostats.tracing.pruning import prune_skeleton
+from topostats.tracing.skeletonize import getSkeleton
+
+LOGGER = logging.getLogger(LOGGER_NAME)
+
+
+class disorderedTrace:
+    """
+    Calculate disordered traces for a DNA molecule and calculates statistics from those traces.
+
+    2023-06-09 : This class has undergone some refactoring so that it works with a single grain. The `trace_grain()`
```

Comment:
This comment is more appropriate to commit messages. Users who are reading the API are unlikely to be interested in this level of detail. Developers who are would read it in the `git log`.

Reply:
Was a remenant from dnatracing. Will remove.